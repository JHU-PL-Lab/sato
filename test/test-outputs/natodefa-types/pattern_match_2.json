[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [],
          "err_match_val": "`A 2",
          "err_match_expected": "Variant `C",
          "err_match_actual": "Variant `A"
        }
      ],
      [
        "Error_match",
        {
          "err_match_aliases": [],
          "err_match_val": "`A 2",
          "err_match_expected": "Variant `B",
          "err_match_actual": "Variant `A"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "match `A 2 with | `B b -> b | `C c -> c end",
    "err_steps": 144
  }
]
