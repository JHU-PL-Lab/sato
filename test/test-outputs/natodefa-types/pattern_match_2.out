Type errors for:
- Input sequence : []
- Found at expr  : match `A 2 with | `B b -> b | `C c -> c
end
--------------------
* Value    : `A 2
* Expected : Variant `C
* Actual   : Variant `A
--------------------
* Value    : `A 2
* Expected : Variant `B
* Actual   : Variant `A
2 errors found. No further control flows exist.
